ztp/extra-manifests-builder/04-accelerated-container-startup/accelerated-container-startup.sh

```diff
@@ -144,31 +144,33 @@ steadystate() {
     echo "last:$last current:$current Waiting to reach $STEADY_STATE_MINIMUM before checking for steady-state"
     return 1
   fi
-  within $last $current $STEADY_STATE_THRESHOLD
+  within "$last" "$current" "$STEADY_STATE_THRESHOLD"
 }
 
 waitForReady() {
   logger "Recovery: Waiting ${MAXIMUM_WAIT_TIME}s for the initialization to complete"
-  local lastSystemdCpuset="$(currentAffinity 1)"
-  local lastDesiredCpuset="$(unrestrictedCpuset)"
+  local lastSystemdCpuset lastDesiredCpuset
+  lastSystemdCpuset=$(currentAffinity 1)
+  lastDesiredCpuset=$(unrestrictedCpuset)
   local t=0 s=10
   local lastCcount=0 ccount=0 steadyStateTime=0
   while [[ $t -lt $MAXIMUM_WAIT_TIME ]]; do
     sleep $s
     ((t += s))
     # Re-check the current affinity of systemd, in case some other process has changed it
-    local systemdCpuset="$(currentAffinity 1)"
+    local systemdCpuset desiredCpuset
+    systemdCpuset="$(currentAffinity 1)"
     # Re-check the unrestricted Cpuset, as the allowed set of unreserved cores may change as pods are assigned to cores
-    local desiredCpuset="$(unrestrictedCpuset)"
-    if [[ $systemdCpuset != $lastSystemdCpuset || $lastDesiredCpuset != $desiredCpuset ]]; then
+    desiredCpuset="$(unrestrictedCpuset)"
+    if [[ $systemdCpuset != "$lastSystemdCpuset" || $lastDesiredCpuset != "$desiredCpuset" ]]; then
       resetAffinity "$desiredCpuset"
       lastSystemdCpuset="$(currentAffinity 1)"
       lastDesiredCpuset="$desiredCpuset"
     fi
 
     # Detect steady-state pod count
-    ccount=$(crictl ps | wc -l)
-    if steadystate $lastCcount $ccount; then
+    ccount=$(crictl ps 2>/dev/null | wc -l)
+    if [[ $ccount -gt 0 ]] && steadystate "$lastCcount" "$ccount"; then
       ((steadyStateTime += s))
```

Comment:
This is the only function change; Hiding the stderr output from crictl and ignoring the result if the total number of running containers is 0